### PR TITLE
Allow custom XBAR latency configurations (e.g., vivado)

### PR DESCRIPTION
**Pull request by micprog**

### CODE REVIEW

**Review by FrancescoConti — APPROVED** [review]
I think this makes sense. @thommythomaso is the maintainer of this IP right? I'd wait for him to merge.

**Review by thommythomaso — APPROVED** [review]
:+1:

**Pull request merged**
